tests/tfhe_rust_bool/ops.mlir

```diff
@@ -26,4 +27,24 @@ module {
     return
   }
 
+  // CHECK-LABEL: func @test_packed_and
+  func.func @test_packed_and(%bsks : !bsks, %lhs : tensor<4x!eb>, %rhs : tensor<4x!eb>) {
```

Comment:
If all you're doing is checking syntax, you can just make the arguments to and_packed the same as the function arguments and skip the setup.